src/GraphQL.Conventions/GraphQL.Conventions.csproj

```diff
@@ -29,14 +27,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL" Version="7.0.0" />
-    <PackageReference Include="GraphQL.DataLoader" Version="7.0.0" />
-    <PackageReference Include="GraphQL.NewtonsoftJson" Version="7.0.0" />
-    <PackageReference Include="GraphQL.SystemReactive" Version="4.8.0" />
-    <PackageReference Include="System.Reflection" Version="4.3.0" />
-    <PackageReference Include="System.Runtime" Version="4.3.1" />
-    <PackageReference Include="System.Reflection.Extensions" Version="4.3.0" />
-    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
+    <PackageReference Include="GraphQL" Version="7.2.0" />
+    <PackageReference Include="GraphQL.DataLoader" Version="7.2.0" />
+    <PackageReference Include="GraphQL.NewtonsoftJson" Version="7.2.0" />
```

Comment:
🤔 strongly coupled with NSJ?

Reply:
The Conventions project has its own server; deserializing requests and serializing responses.  It is not within the scope of this PR to remove or modify such behavior.

Reply:
Just a question.